.eslintrc.json

```diff
@@ -4,7 +4,8 @@
     "prettier/prettier": ["error"],
     "no-use-before-define": "off",
     "prefer-destructuring": "warn",
-    "consistent-return": "off"
+    "consistent-return": "off",
+    "no-console": ["error"]
```

Comment:
it would be good to understand why you need to expicitly put this in.  We have a no-console rule in notm, but haven't had to put it in .eslintrc.  What other rules are missing?